tests/ui/parser/variadic-ffi-semantic-restrictions.rs

```diff
@@ -83,3 +83,8 @@ trait T {
     //~^ ERROR only foreign, `unsafe extern "C"`, or `unsafe extern "C-unwind"` functions may have a C-variadic arg
     //~| ERROR `...` must be the last argument of a C-variadic function
 }
+
+unsafe extern "C" {
+    safe fn s_f1(...);
```

Comment:
Given that this is machine applicable, it would be good to have one or two examples with longer signatures containing additional arguments. A printf-style signature would be a good example.

Reply:
(And how did I get this far without knowing that `safe` was a keyword?)